votes/0593-hfxsd-as-docs-reviewer.md

```diff
@@ -0,0 +1,31 @@
+# A Vote for hfxsd as Docs Reviewer
+
+## Proposal
+
+[@hfxsd](https://github.com/hfxsd) has been working on TiDB documentation for the past three months. His work covers the documentation of TiDB in both Chinese and English. Also, he gave a lot of good suggestions in his review comments. I think @hfxsd shows his qualification as a reviewer of the Docs Team. The details of his contributions are listed as follows:
+
+- Authored pull requests
+    - [docs](https://github.com/pingcap/docs/pulls?q=is%3Apr+is%3Amerged+author%3Ahfxsd): 12
+    - [docs-cn](https://github.com/pingcap/docs-cn/pulls?q=is%3Apr+is%3Amerged+author%3Ahfxsd): 7
+    - [docs-dm](https://github.com/pingcap/docs-dm/pulls?q=is%3Apr+is%3Amerged+author%3Ahfxsd): 1
+- Reviewed pull requests
+    - [docs](https://github.com/pingcap/docs/pulls?q=is:pr+reviewed-by:hfxsd+is:merged+-author:ti-chi-bot): 9
+    - [docs-cn](https://github.com/pingcap/docs-cn/pulls?q=is:pr+reviewed-by:hfxsd+is:merged+-author:ti-chi-bot): 9
+    - [docs-dm](https://github.com/pingcap/docs-dm/pulls?q=is:pr+reviewed-by:hfxsd+is:merged+-author:ti-chi-bot): 8
+- Resolved issues
+    - docs: [#5457](https://github.com/pingcap/docs/issues/5457), [#6541](https://github.com/pingcap/docs/issues/6541)
+    - docs-cn: [#7337](https://github.com/pingcap/docs-cn/issues/7337)
+
+I (@lilin90) hereby nominate @hfxsd as Docs Reviewer and call for a vote.
+
+## Deadline
+
+The vote will be open for at least 3 days unless there is an objection or not enough votes.
+
+## Scope
+
+* Team Docs
```

Comment:
Docs Team?

Reply:
@qiancai I also wanted to use "Docs Team", but in the existing files in [votes](https://github.com/pingcap/community/tree/master/votes), the "Team xxx" style is used. Is this on purpose? @tisonkun

Reply:
No. It's a suggestion. You may change if you think "Docs Team" better.

Reply:
> No. It's a suggestion. You may change if you think "Docs Team" better.

@tisonkun Got it. Or we can use "Team: xxx".